backend/apps/notification/services.py

```diff
@@ -0,0 +1,35 @@
+from django.db import transaction
+from .models import Notification
+
+
+
+def create_notification(user, notification_type, content, is_important=False):
+    
+    notification = Notification(
+        user=user,
+        notification_type=notification_type,
+        content=content,
+        is_important=is_important,
+    )
+    notification.save()
+    return notification
+
+
+
+def mark_notification_as_read(notification):
```

Comment:
Instead, just create a basic notification_update, which can accept parameters, see communities app for example.